server/WebAPI/Services/SerialsService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Entities.DataAccess;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPI.DTO;
+
+namespace WebAPI.Services
+{
+    public interface ISerialsService
+    {
+        List<SerialDTO> GetSerials();
+    }
```

Comment:
interface in separate file
Services folder - interfaces in root
Services/Implementation - the interface implementations